changelog/0032-upgrade-to-elasticsearch-2.4.6.yml

```diff
@@ -1,102 +1,115 @@
 ---
 title: ES upgrade from 1.7.6 to 2.4.6
-key: upgrade-elasticsearch
+key: upgrade-to-elasticsearch-2.4.6
 date: 2020-02-05
 optional_per_env: no
 min_commcare_version: 064bb6dfc2b333cc8e6e3955b0c5d30513c5f4be
 context: |
   This change upgrade Elasticsearch from 1.7.6 to 2.4.6 version.
+  CommCare HQ releases after April 2, 2020 will not continue to support Elasticsearch 1.7.6,
+  so we strongly recommend applying this change before then.
 
 details: |
   As part of our ongoing effort to keep CommCare HQ up to date with the latest tools and
   libraries we have updated Elasticsearch from 1.7.6 to 2.4.6 version.
 
 update_steps: |
 
+  Before you start, make sure that your disk usage is well below 50%,
+  or that you have a place to back up to on another disk or remotely.
+
+  0. For commands including `${ENV}` below to work, make sure to set `ENV` to the name of your environment
+  ```
+  ENV=${ENV}
+  ```
```

Comment:
Could also add to this something like
```
ES_HOST=$(cchq ${ENV} lookup elasticsearch:0)
```
That'd make the below commands a bit easier to follow if something goes awry

Reply:
Good idea, ✔️